frontend/apps/design-system-storybook/package.json

```diff
@@ -5,10 +5,12 @@
     "code.org"
   ],
   "scripts": {
+    "clean": "rimraf storybook-static",
     "dev": "yarn run storybook",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "test-storybook": "test-storybook",
+    "test:ui": "bash ./scripts/test-storybook.sh",
+    "test-storybook": "test-storybook --url http://localhost:6006",
```

Comment:
The difference between these two commands seems non-intuitive

Reply:
Rename `test:ui` to `test:ui:ci` to clarify its usage in CI environments and `test-storybook` to `test:ui:local` to clarify its intended purpose to locally test.